### ISSUE TITLE
"Ticket is missing its ticket type ID" when saving an Event 

### ISSUE THREAD

**Issue by pixelmachine** [verbb/Events#10]
I've just installed the trial and I can't seem to save any events with tickets added. They save fine without.

```
yii\base\InvalidConfigException: Ticket is missing its ticket type ID in /Users/Name/Sites/site/craft/vendor/verbb/events/src/elements/Ticket.php:268
Stack trace:
#0 /Users/Name/Sites/site/craft/vendor/verbb/events/src/elements/Ticket.php(172): verbb\events\elements\Ticket->getType()
#1 /Users/Name/Sites/site/craft/vendor/yiisoft/yii2/base/BaseObject.php(109): verbb\events\elements\Ticket->init()
#2 /Users/Name/Sites/site/craft/vendor/verbb/events/src/helpers/EventHelper.php(28): yii\base\BaseObject->__construct()
#3 /Users/Name/Sites/site/craft/vendor/verbb/events/src/elements/Event.php(385): verbb\events\helpers\EventHelper::populateEventTicketModel(Object(verbb\events\elements\Event), Array, 'new1')
#4 /Users/Name/Sites/site/craft/vendor/verbb/events/src/helpers/EventHelper.php(104): verbb\events\elements\Event->setTickets(Array)
#5 /Users/Name/Sites/site/craft/vendor/verbb/events/src/controllers/EventsController.php(161): verbb\events\helpers\EventHelper::populateEventFromPost()
#6 [internal function]: verbb\events\controllers\EventsController->actionSave()
#7 /Users/Name/Sites/site/craft/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#8 /Users/Name/Sites/site/craft/vendor/yiisoft/yii2/base/Controller.php(157): yii\base\InlineAction->runWithParams(Array)
#9 /Users/Name/Sites/site/craft/vendor/craftcms/cms/src/web/Controller.php(109): yii\base\Controller->runAction('save', Array)
#10 /Users/Name/Sites/site/craft/vendor/yiisoft/yii2/base/Module.php(528): craft\web\Controller->runAction('save', Array)
#11 /Users/Name/Sites/site/craft/vendor/craftcms/cms/src/web/Application.php(297): yii\base\Module->runAction('events/events/s...', Array)
#12 /Users/Name/Sites/site/craft/vendor/craftcms/cms/src/web/Application.php(565): craft\web\Application->runAction('events/events/s...', Array)
#13 /Users/Name/Sites/site/craft/vendor/craftcms/cms/src/web/Application.php(281): craft\web\Application->_processActionRequest(Object(craft\web\Request))
#14 /Users/Name/Sites/site/craft/vendor/yiisoft/yii2/base/Application.php(386): craft\web\Application->handleRequest(Object(craft\web\Request))
#15 /Users/Name/Sites/site/craft/web/index.php(21): yii\base\Application->run()
#16 {main}
```

**Comment by engram-design** [comment]
Have you selected a ticket type for all your tickets> Do you have a screenshot of what you're trying to save when this error is thrown?

**Comment by pixelmachine** [comment]
Yep and sure:

![Screen Shot 2019-05-16 at 17 28 09](https://user-images.githubusercontent.com/1082223/57888411-1c33ca00-7800-11e9-929c-afe21d0fb40d.png)

**Comment by pixelmachine** [comment]
If I remove the ticket, it'll save fine.

**Comment by engram-design** [comment]
Should be fixed in [1.1.1](https://github.com/verbb/events/releases/tag/1.1.1)
